Source/Particles/Resampling/LevelingThinning.cpp

```diff
@@ -58,8 +63,8 @@ void LevelingThinning::operator() (WarpXParIter& pti, const int lev,
             const auto cell_stop  = static_cast<int>(cell_offsets[i_cell+1]);
             const int cell_numparts = cell_stop - cell_start;
 
-            // do nothing for cells without particles
-            if (cell_numparts == 0)
+            // do nothing for cells with less particles than min_ppc
```

Comment:
Since this also prevents a div/0 it is potentially worth to keep the original meaning documented, too:
```suggestion
            // do nothing for cells with less particles than min_ppc
            // (this intentionally includes skipping empty cells, too)
```
(make the code more stable with respect to future refactorings)